include/evolution_graphics.h

```diff
@@ -10,7 +10,7 @@ u8 LaunchTask_PreEvoSparklesSet2(void);
 u8 LaunchTask_PostEvoSparklesSet1(void);
 u8 LaunchTask_PostEvoSparklesSet2AndFlash(u16 species);
 u8 LaunchTask_PostEvoSparklesSet2AndFlash_Trade(u16 species);
-u8 sub_817C3A0(u8 preEvoSpriteID, u8 postEvoSpriteID);
+u8 LaunchTask_PrePostEvoMonSprites(u8 preEvoSpriteId, u8 postEvoSpriteId);
 
 #define EvoGraphicsTaskEvoStop  data[8]
```

Comment:
does this mean the same as tBButtonCancel?

Reply:
yes